### PR TITLE
Updated to use "Hello World" example

### PR DESCRIPTION
**Pull request by CarrieBrown**
Pi example moved to ScalingUp-R

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,31 +1,25 @@
...
+
 [TOC]
 
 ## Overview
-This tutorial describes how to compute the value of pi using the R statistical package on the OSG. For this example, we'll estimate the value of pi using a Monte Carlo method. We'll first run the program locally as a test.  After that we'll create a submit file, submit it to OSG using OSG Connect, and then collate results when the jobs finish.
-
-
-### Background
-Some background is useful here. We define a square inscribed by a unit circle. We randomly sample points, and calculate the ratio of the points outside of the circle to the points inside for the first quadrant. This ratio approaches pi/4.
+This tutorial describes how to run R scripts on the OSG. We'll first run the program locally as a test.  After that we'll create a submit file, submit it to OSG using OSG Connect, and look at the results when the jobs finish. Finally, we will talk about how to use custom R libraries on OSG Connect.
```
Comment (ChristinaLK) [inline README.md]:
```suggestion
This tutorial describes how to run R scripts on the OSG. We'll first run the program locally as a test.  After that we'll create a submit file, submit it to OSG using OSG Connect, and look at the results when the jobs finish.
```

File: README.md [hunk 2]
```diff
@@ -46,110 +40,87 @@ Once we have the path set up, we can try to run R. Don't worry if you aren't an
...
-That's pretty close! With even more sample sets — that is, more Queue jobs in the cluster — we can statistically come even closer.
-
-##What to do next?
-The `R.submit` file may have included a few lines that you are unfamiliar with.  For example, `$(Cluster)` and `$(Process)` are variables that will be replaced with the job's cluster and process id.  This is useful when you have many jobs submitted in the same file.  Each output and error file will be in a separate directory.
-
-Also, did you notice the transfer_input_files line?  This tells HTCondor what files to transfer with the job to the worker node.  You don't have to tell it to transfer the executable, HTCondor is smart enough to know that the job will need that.  But any extra files, such as our MonteCarlo R file, will need to be explicitly listed to be transferred with the job.  You can use transfer_input_files for input data to the job, as shown in [Transferring data with HTCondor](https://github.com/OSGConnect/tutorial-htcondor_transfer).
+	$ cat R.out.3796250.0
+	[1] "Hello World!"
 
 ## Getting Help
```
Comment (ChristinaLK) [inline README.md:133]: Maybe add links to the other guides -- submitting multiple jobs + using libraries?

**Review by ChristinaLK — COMMENTED** [review]
:+1:

File: README.md (hunk 2 above)
Comment (CarrieBrown) [reply README.md:133]: 👍Added links to other R guides!

**Pull request merged**